tests/fsharpqa/Source/Optimizations/GenericComparison/Hash02.il.bsl

```diff
@@ -59,8 +59,6 @@
     {
       // Code size       1 (0x1)
       .maxstack  8
-      .language '{AB4F38C9-B6E6-43BA-BE3B-58080B2CCCE3}', '{994B45C4-E6E9-11D2-903F-00C04FA302A1}', '{5A869D0B-6611-11D3-BD2A-0000F80849BD}'
-      .line 7,8 : 8,29 'C:\\GitHub\\dsyme\\fsharp\\tests\\fsharpqa\\source\\Optimizations\\GenericComparison\\Hash02.fsx'
```

Comment:
This is an odd test - the entire function actually gets optimised away and now the breakpoint disappears in release code too, which is ok really (doesn't matter much either way). The source code was this
```
    let f4_triple() = 
       let mutable x = 1
       let t3 = (1,2,3)
       for i = 0 to 10000000 do
           hash t3 |> ignore
```

Reply:
Is the function being optimised in debug builds as well? I'd expect users to be able to step through this code even if it doesn't make much sense.